c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po

```diff
@@ -369,23 +369,23 @@ msgstr "Mode lecture écriture"
 
 #: ./c2cgeoportal/views/entry.py:83
 msgid "title i18n"
-msgstr "Test de l'i18n"
+msgstr "Test de l’i18n"
 
 #: ./c2cgeoportal/views/entry.py:130
 msgid ""
 "WARNING! an error occured while trying to read the mapfile and recover the "
 "themes."
-msgstr "ATTENTION ! Une erreur s'est produite en essayant de lire le mapfile  et de récupérer les thèmes."
+msgstr "ATTENTION ! Une erreur s’est produite en essayant de lire le mapfile  et de récupérer les thèmes."
```

Comment:
Not your fault but can you correct that:
> ...le mapfile__et de...

Dobble spaces between 'mapfile' and 'et'